client/go/internal/cli/cmd/visit_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestQuoteFunc(t *testing.T) {
```

Comment:
We should add a test for the visiting part. Something similar is done in https://github.com/vespa-engine/vespa/blob/master/client/go/internal/cli/cmd/document_test.go.